Tests/MapperTests/RawRepresentibleValueTests.swift

```diff
@@ -130,6 +130,38 @@ final class RawRepresentibleValueTests: XCTestCase {
         XCTAssertNil(test.value)
     }
 
+    func testConvertibleArrayOfKeysDoesNotThrow() {
+        struct Test: Mappable {
+            let value: Value?
+            init(map: Mapper) throws {
+                try self.value = map.from(["a", "b"])
+            }
+        }
+
+        enum Value: String {
+            case first = "hi"
+        }
+
+        let test = try? Test(map: Mapper(JSON: ["a": 1, "b": "hi"]))
+        XCTAssertNotNil(test?.value)
+    }
+
+    func testConvertibleArrayOfKeysThrowsWhenMissing() {
+        struct Test: Mappable {
+            let value: Value?
+            init(map: Mapper) throws {
+                try self.value = map.from(["a", "b"])
+            }
+        }
+
+        enum Value: String {
+            case first = "hi"
+        }
+
+        let test = try? Test(map: Mapper(JSON: ["a": 1, "b": 2]))
+        XCTAssertNil(test?.value)
```

Comment:
should just test that `test` is nil and make `value` non-optional

Reply:
good call, done